docs/deployment/benchmark.md

```diff
@@ -0,0 +1,65 @@
+# Benchmarking your Invenio instance
+
+## Populate your instance
+
+!!! info "Already populated instances"
+    If you already have content in the instance this step is not needed.
+
+We are going to create some demo records using a new CLI command. We are going
+to use `invenio` commands. You can launch a job using the
+[provided job templates](../jobs) (recommended) or execute them in a web
+container's terminal.
+
+!!! warining "OpenShift commands"
+    The following commands assume you are using OpenShift, for bare Kubernetes,
+    commands might be slightly different.
+
+**Launch a job**
+
+The following job will be executed by a worker and create 10 demo records.
+
+``` console
+$ oc process -f job.yml --param JOB_NAME='demo-data' \
+  --param JOB_COMMAND='invenio rdm-records demo' | oc create -f -
+```
+
+**Execution in a web container**
+
+To populate the instance with the alternative method, you need to connect to a
+running pod, activate python3 and execute the command:
+
+```console
+$ oc get pods # List all pods
+$ oc exec -it <web-pod-id> bash # Connect to a web pod
+$ . scl_source enable rh-python36 # Activate python3
+$ invenio rdm-records demo # Create 10 demo records
+```
+
+This methods populate the instance with 10 records. If you need more, you can
+run it multiple times. However, obtaining a high number of records might be
+time consuming with this method. In that case, you can use bulk indexing, you
+can find more documentation on how to do it [here](https://github.com/inveniosoftware/helm-invenio/tree/master/benchmark#populate-your-instance).
+
+## Benchmark
+
+In order to test the load that your instance is able to stand, you can use
+[Locust](https://docs.locust.io/en/stable/). [Here](https://github.com/inveniosoftware/helm-invenio/blob/master/benchmark/locustfile.py)
+you can find a `locust.py` file ready to test an Invenio instance. Let's
+test it!
```

Comment:
Normal/Question: How do we use that file? We need to download it somewhere or we need to clone that repository and go in a directory?

Reply:
Nice point, added :)